[2021-12-20 09:36:24,773]-[object.py]-[<module>]-[300]-12s: [INFO]-8s>> ces
[2021-12-20 09:36:24,777]-[object.py]-[<module>]-[301]-12s: [INFO]-8s>> hello
[2021-12-20 09:59:44,717]-[object.py]-[<module>]-[300]-12s: [INFO]-8s>> ces
[2021-12-20 09:59:44,719]-[object.py]-[<module>]-[301]-12s: [INFO]-8s>> hello
[2021-12-20 13:30:39,205]-[CasSystemScrapy.py]-[<module>]-[598]-12s: [INFO]-8s>> ces
[2021-12-20 13:30:39,514]-[CasSystemScrapy.py]-[<module>]-[599]-12s: [INFO]-8s>> hello
[2021-12-20 13:30:53,177]-[CasSystemScrapy.py]-[<module>]-[598]-12s: [INFO]-8s>> ces
[2021-12-20 13:30:53,178]-[CasSystemScrapy.py]-[<module>]-[599]-12s: [INFO]-8s>> hello
[2021-12-20 13:31:24,362]-[CasSystemScrapy.py]-[<module>]-[598]-12s: [INFO]-8s>> ces
[2021-12-20 13:31:24,620]-[CasSystemScrapy.py]-[<module>]-[599]-12s: [INFO]-8s>> hello
[2021-12-20 13:32:11,419]-[CasSystemScrapy.py]-[<module>]-[598]-12s: [INFO]-8s>> ces
[2021-12-20 13:32:11,694]-[CasSystemScrapy.py]-[<module>]-[599]-12s: [INFO]-8s>> hello
[2021-12-20 13:35:40,398]-[CasSystemScrapy.py]-[<module>]-[598]-12s: [INFO]-8s>> ces
[2021-12-20 13:35:40,399]-[CasSystemScrapy.py]-[<module>]-[599]-12s: [INFO]-8s>> hello
[2021-12-20 13:46:07,131]-[CasSystemScrapy.py]-[<module>]-[598]-12s: [INFO]-8s>> ces
[2021-12-20 13:46:07,132]-[CasSystemScrapy.py]-[<module>]-[599]-12s: [INFO]-8s>> hello
[2021-12-20 13:47:57,304]-[CasSystemScrapy.py]-[<module>]-[598]-12s: [INFO]-8s>> ces
[2021-12-20 13:47:57,305]-[CasSystemScrapy.py]-[<module>]-[599]-12s: [INFO]-8s>> hello
[2021-12-20 13:48:34,228]-[CasSystemScrapy.py]-[<module>]-[598]-12s: [INFO]-8s>> ces
[2021-12-20 13:48:34,229]-[CasSystemScrapy.py]-[<module>]-[599]-12s: [INFO]-8s>> hello
[2021-12-20 13:49:46,440]-[CasSystemScrapy.py]-[<module>]-[598]-12s: [INFO]-8s>> ces
[2021-12-20 13:49:46,441]-[CasSystemScrapy.py]-[<module>]-[599]-12s: [INFO]-8s>> hello
[2021-12-20 13:54:24,611]-[CasSystemScrapy.py]-[get_apply_data]-[581]-12s: [ERROR]-8s>> 解析明细信息出错，错误单号：GA-202111-1：Unexpected Error: list index out of range
